lib/reek/cli/report/report.rb

```diff
@@ -1,5 +1,7 @@
 require 'rainbow'
 require 'json'
+require_relative 'formatter'
+require_relative 'heading_formatter'
```

Comment:
Already fixed in https://github.com/troessner/reek/pull/540/files#diff-9eaec13b838dd3d833fd276a922f1ac5R3